packages/client/src/demo/App.tsx

```diff
@@ -252,6 +257,7 @@ const ExampleCsvSeries = () => {
           "geoId/0668000",
           "geoId/0653000",
         ],
+        fieldDelimiter: "__",
```

Comment:
Any particular reason to use "__" as the delimiter in the demo? Assuming this will be code people will copy and paste from (and eventually email us questions about), it might be cleaner to use the new default "." delimiter.

Reply:
This is actually a nuance with the vega lite library and how it handles "." characters in the csv header. Added a comment to clarify.